### PR TITLE
Include a standard mvc app in start up test

### PR DESCRIPTION
**Pull request by troydai**
@rynowak @kirthik @muratg @davidfowl

### CODE REVIEW

File: testapp/StandardMvc/ApiControllers/ValuesController.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc;
+
+namespace Microsoft.AspNet.Test.Perf.WebFx.Apps.HelloMvc.ApiControllers
```
Comment (rynowak) [inline testapp/StandardMvc/ApiControllers/ValuesController.cs]: That's quite a namespace you've got there. Come here often?

**Comment by troydai** [comment]
/cc @dougbu @rynowak 

Please help review the sample `StandardMvc`.

File: testapp/StandardMvc/Startup.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                routes.MapRoute(
+                    "controllerActionRoute",
+                    "{controller}/{action}",
+                    new { controller = "Home", action = "Index" });
+
+                routes.MapRoute(
+                    "controllerRoute",
+                    "{controller}",
+                    new { controller = "Home" });
+            });
```
Comment (rynowak) [inline testapp/StandardMvc/Startup.cs]: Remove this route, we don't recommend using conventional routing this way

**Comment by rynowak** [comment]
I thought we had already built a 'standard mvc' app? is this just porting to a new repo?

**Comment by troydai** [comment]
@rynowak yes. The start up tests will run here. The app is ported here.

File: testapp/StandardMvc/ApiControllers/ValuesController.cs (hunk 1 above)
Comment (troydai) [inline testapp/StandardMvc/ApiControllers/ValuesController.cs]: I'll replace it with a shorter name.

**Comment by rynowak** [comment]
Is there a better way to share this code? (submodule perhaps) Is the other repo being phased out?

File: testapp/StandardMvc/project.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+    "commands": {
+        "web": "Microsoft.AspNet.Hosting server=Microsoft.AspNet.Server.WebListener server.urls=http://localhost:5000"
+    },
+    "dependencies": {
+        "Microsoft.AspNet.Mvc": "6.0.0-*",
+        "Microsoft.AspNet.StaticFiles": "1.0.0-*",
+        "Microsoft.AspNet.Server.IIS": "1.0.0-*",
+        "Microsoft.AspNet.Server.WebListener": "1.0.0-*",
+        "Microsoft.Framework.Runtime": "1.0.0-*",
```
Comment (davidfowl) [inline testapp/StandardMvc/project.json]: What?

File: testapp/StandardMvc/project.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        "Microsoft.Framework.ConfigurationModel": "1.0.0-*",
```
Comment (davidfowl) [inline testapp/StandardMvc/project.json]: You can get rid of this

**Pull request merged**